### ISSUE TITLE
Docker "stats" under CC run doesn't work properly

### ISSUE THREAD

**Issue by bktan8** [clearcontainers/runtime#200]
The only stat that updates is the NET I/O.  All other stats (CPU %, MEM USAGE, MEM %, BLOCK I/O, PIDS) are 0.

**Comment by dlespiau** [comment]
Indeed. Those stats depend on having cgroups for the containerized workload, which CC replaces by a VM. Note that, in the future, we'll probably add crgoups inside the VM and will be then able to obtain those stats. It's still a bit far out though.

**Comment by vinay-y** [comment]
Hi, I am using cc-oci-runtime version: 2.2.4.
As there are no cgroups, we are unable to monitor cpu/memory utilization of the containers from host machine. Is there any way of monitoring this? One way is to run a daemon within the container which the host can request to give stats. But, can we do it somehow from the host directly? Using cc-shim or something? Can we do this in runtime 3.0?

Thanks.

**Comment by grahamwhaley** [comment]
Hi @vinay-y 
If you are looking for just the overall total CPU and memory usage of the container, then you can examine the values for the VM that is holding the container (qemu), and that will give you the total cpu and memory that that container is consuming.
Using the `top` command host side (or any other host side memory and cpu analysis tools) should give you a similar set of data to `docker stats`.
We are adding feature support to CC3 (such as the 'docker ps' pull request currently pending), but we are in a stabilisation phase heading for release right now, so are pretty much in feature freeze at present.
Of course, any pull requests adding new features are always more than welcome ;-)
